Chandra/cmd_states/cmd_states.py

```diff
@@ -104,9 +105,22 @@ def add_trans(date=date, **kwargs):
         # if no key in kwargs is in the exclude set then update transition
         if not (exclude and set(exclude).intersection(kwargs)):
             transitions.setdefault(date, {}).update(kwargs)
+            transitions['last_date'] = date
```

Comment:
I think this approach is good.  I would suggest adding a comment on why exactly this is needed.  This might be a long-ish comment explaining how the `GET_PITCH` command works and interacts with maneuvers.